Backend/src/main/java/Digitact/Backend/Model/User/Applicant.java

```diff
@@ -74,4 +80,10 @@ public Positions getPositions() {
     public void setPositions(Positions positions) {
         this.positions = positions;
     }
+
+    public void addKeyCompetence(KeyCompetence keyCompetence) { keyCompetencies.add(keyCompetence); }
+
+    public List<KeyCompetence> getKeyCompetencies() { return new ArrayList<KeyCompetence>(keyCompetencies); }
+
+    public void addKeyCompetencies(List<KeyCompetence> keyCompetencies) { keyCompetencies.forEach(comp -> this.keyCompetencies.add(comp)); }
```

Comment:
Its always good to maintain consistency in method naming. So, it would be better to change "addkey" to "setKey".

Reply:
To me, a method named "set" always sounds like it **overrides** the current content with what is being passed as parameter. This is not the case here: It appends stuff, so the old content is not being removed. Because of this, I wasn't really comfortable with using "set". What do you think about this?

I guess we could do one of the following two things:
1. I just rename it to "set". And we keep in mind that we should refactor this later.
2. I rename all "set..." methods to "add..." or "append..." or whatever.

What would you prefer?

Reply:
The word set in OOP means to declare&change a value to a variable, while add means to insert an element to a list for example. 
Usually, "add" used to add an object to another object. 
For example: 
"SetList" means, myList= newList; // old list is removed
"AddList" means myList += newList; // extend my old list  .

Reply:
FYI: 
usually, the "key" should be final, which means its only set by initialization through the constructor. so we don't need a set method. :)

Reply:
I'll review the code and refactor it today!

Reply:
> The word set in OOP means to declare&change a value to a variable, while add means to insert an element to a list for example.

OK, so my implementation should be fine, right? If not, then please elaborate a bit more on what I should change and in which way. :) 

> usually, the "key" should be final

Which key are you referring to?

> I'll review the code and refactor it today!

Could we merge my PR before you refactor anything? Otherwise, I will get merge conflicts on that one.

Reply:
I'll check the code and fix the "bugs/errors" if there is and will let u know what I changed and why @MBuchalik  :)

Reply:
I didn't even know you have write access on my repo :D